scripts/repoHandlers.js

```diff
@@ -106,6 +106,48 @@ function updateTrackedReposInOrgGitHub(trackedRepos, websiteUrl) {
 	})
 }
 
+/**
+ * Updates the user's GitHub repositories page to visually indicate which repositories are being tracked.
+ * 
+ * @param {Array} trackedRepos - List of tracked repositories.
+ * @param {string} websiteUrl - The URL of the website.
+ */
+function updateTrackedReposInUserGitHub(trackedRepos, websiteUrl) {
+	const allUserRepo = document.getElementById('user-repositories-list');
```

Comment:
I think this is the only line that is different between this function and the `updateTrackedReposInOrgGitHub` function.
If it is, maybe you can just pass a single parameter (called `ownerType`, taking "user" or "org" as allowed values) to a common `updateTrackedReposGitHub` function, and assign this variable using a ternary operation.

```suggestion
	const allRepos = (ownerType === "user") ? document.getElementById('user-repositories-list') : document.getElementById("org-repositories");
```